lib/block/blockdownload.js

```diff
@@ -205,12 +203,10 @@ class BlockDownload extends EventEmitter {
             }
 
             if (randomFileAccess) {
-                console.log("Block download: closing random file accessor");
```

Comment:
these logs show up in every run: https://adobeiort.splunkcloud.com/en-US/app/search/search?q=search%20index%3D%22runtime_prod_ext%22%20activation_id%3Df94487dd4ddd4fe98487dd4ddddfe9c6%20%7C%20sort%20_time%20%7C%20fields%20%2B%20_time%2Clog_message%20%7C%20fields%20-%20_raw%20%7C%20outputtext&display.page.search.mode=smart&dispatch.sample_ratio=1&earliest=%40d&latest=now&display.page.search.showFields=0&display.events.type=table&display.page.search.tab=statistics&display.general.type=statistics&sid=1641855681.306221
to avoid clogging the logs, shorten to just the "when its over log". if we don't see the log on line 207, then we know something went wrong awaiting file random file access